<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiVoicePlayerWidget</class>
 <widget class="QWidget" name="GuiVoicePlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>42</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>248</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="mp_pbPlay">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>0</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../beebeep.qrc">
       <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="mp_sliderPosition">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mp_lDuration">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_pbPlay</tabstop>
  <tabstop>mp_sliderPosition</tabstop>
 </tabstops>
 <resources>
  <include location="../beebeep.qrc"/>
 </resources>
 <connections/>
</ui>
